### PR TITLE
feat(LTAK-194): prepare Coding Dojo

### PR DESCRIPTION
**Pull request by vincnt28**

### CODE REVIEW

File: src/app/sections/landing-pages/coding-dojo/coding-dojo.component.ts [hunk 1]
```diff
@@ -16,15 +18,15 @@ export class CodingDojoComponent implements OnInit {
     },
     title: 'Coding Dojo - Was ist das?',
     subtitle:
-      'Erlebe die Vielfalt technologischer Möglichkeiten hautnah beim Coding Dojo. Hier lernst Du alles, was Du über Digitalisierung und Webentwicklung wissen musst, um Dich im Netz sicher zu bewegen sogar Deine eigenen Programme zu bauen. <br><br> Vom ersten Einstiegsworkshops über Grundlagen und Erweiterungskurse bei uns oder in Schulen ist alles dabei. <br><br> Du willst mitmachen? Dann können Du oder Deine Eltern uns einfach eine Anfrage schicken. Wir freuen uns auf Euch!',
+      'Erlebe die Vielfalt technologischer Möglichkeiten hautnah beim Coding Dojo. Hier lernst Du alles, was Du über Digitalisierung und Webentwicklung wissen musst, um Dich im Netz sicher zu bewegen und sogar Deine eigenen Programme zu bauen. <br><br> Vom ersten Einführungskurs über Dein Training bei uns bis hin zu Aktionen in Schulen (AGs) ist alles dabei.<br><br> Du willst mitmachen? Dann können Du oder Deine Eltern uns einfach eine Anfrage schicken. Wir freuen uns auf Euch!',
     // Will be uncommented, when the login for the coding dojo is ready
     // showCodingDojoBtn: true,
```
Comment (vincnt28) [inline src/app/sections/landing-pages/coding-dojo/coding-dojo.component.ts:23]: showCodingDojoBtn must be true, to get the "ANFRAGE SENDEN" and "ZUM CODING DOJO" Button - compare with figma

File: src/app/sections/landing-pages/coding-dojo/coding-dojo.component.ts [hunk 2]
```diff
@@ -16,15 +18,15 @@ export class CodingDojoComponent implements OnInit {
...
+      text: 'Hey, du! Bist du bereit, die Geheimnisse hinter all den Anwendungen, die du täglich nutzt, zu enthüllen? Unser Einführungskurs ist der perfekte Start, um in die aufregende Welt der Programmierung einzutauchen.<br><br> Diesen Kurs musst Du absolvieren, um an unserem regelmäßigen Training teilnehmen zu können. Unsere supernetten Trainer:innen, die echte Profis in der Anwendungsentwicklung sind, werden Dir die Grundlagen zeigen, damit Du im Training voll durchstarten kannst.<br><br> Keine Sorge, es wird total spaßig und interaktiv – langweilige Vorlesungen gehören der Vergangenheit an! Bereite Dich darauf vor, die Apps Deiner Träume zu erschaffen.<br><br> Melde Dich jetzt an und lass uns gemeinsam die faszinierende Welt der App- und Webentwicklung entdecken!<br><br> Falls Du schon unseren Einstiegsworkshop ”Einführung in die App- und Webentwicklung” in der Schule mitgemacht hast, gib uns Bescheid, denn dann bist Du bereits fit fürs Training.',
```
Comment (vincnt28) [inline src/app/sections/landing-pages/coding-dojo/coding-dojo.component.ts:29]: typos, 
1st section: "du" need to be checked, the first three need to start with a uppercase

File: src/app/sections/landing-pages/coding-dojo/coding-dojo.component.ts (hunk 1 above)
Comment (DKoenig9) [reply src/app/sections/landing-pages/coding-dojo/coding-dojo.component.ts:23]: First the login on MM needs to be fixed. For now the buttons will be commented out

**Review by vincnt28 — COMMENTED** [review]
approved

**Review by DKoenig9 — APPROVED** [review]

**Pull request merged**